src/components/map/RouteLayer.js

```diff
@@ -24,8 +24,8 @@ class RouteLayer extends Component {
     });
   };
 
-  componentDidUpdate() {
-    const {mapBounds, stops, setMapBounds = () => {}} = this.props;
+  componentDidMount() {
```

Comment:
didMount is the correct place for this, otherwise the map could get unruly when many updates happen.